[AOTI][XPU] Suppress multi-line comment warning for XPU.

etaf opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #147710

This PR aim to suppress multi-line comment waring in sycl header when building Inductor cpp_wrapper .
```
/intel/oneapi/compiler/2025.0/include/sycl/detail/builtins/builtins.hpp:235:1: warning: multi-line comment [-Wcomment]
``` 


cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov

etaf committed:
[AOTI][XPU] Surpress multi-line comment warning for XPU.

[ghstack-poisoned]

EikanWang commented on torch/_inductor/cpp_builder.py:
```diff
@@ -1221,6 +1221,7 @@ def get_cpp_torch_device_options(
 
     if device_type == "xpu":
         definations.append(" USE_XPU")
+        cflags += ["Wno-comment"]
```
LGTM. By the way, it is better to add some comments and submit a github Issue to the Intel/LLVM community.

etaf replied:
Thanks, I'll report this issue.

etaf committed:
Update on "[AOTI][XPU] Suppress multi-line comment warning for XPU."


This PR aim to suppress multi-line comment waring in sycl header when building Inductor cpp_wrapper .
```
/intel/oneapi/compiler/2025.0/include/sycl/detail/builtins/builtins.hpp:235:1: warning: multi-line comment [-Wcomment]
``` 


cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

etaf commented:
@pytorchbot merge
